ui/components/MesheryPerformance/Dashboard.js

```diff
@@ -113,7 +113,11 @@ function Dashboard({
                   </div>
                 </div>
               </Grid>
-              <div style={{ height : "10.4rem", width : "1px", background : "black", top : "11.2rem", bottom : "0", position : "absolute", left : "36%", backgroundColor : "#36454f", opacity : "0.7" }}></div>
+              <Grid item xs>
+                <div className={classes.paper}>
+                  <div style={{ height : "10.4rem", width : "1px", background : "black", bottom : "0", position : "relative", left : "36%", backgroundColor : "#36454f", opacity : "0.7" }}>

```

Comment:
```suggestion
                  <div style={{ height : "10.4rem", width : "1px", background : "black", bottom : "0", position : "initial", left : "36%", backgroundColor : "#36454f", opacity : "0.7" }}>

```
try using this instead, I'm not sure it would work or not, `relative` is making it look a little eerie.
Also if you could just make divider fixed in one position, while `Results` and `Profiles` remain the same as they are now, in the screen recording you've uploaded they look unaligned.